### DISCUSSION TITLE
Popover with double icon

### DISCUSSION THREAD

**Discussion by Orkuml** [ant-design/ant-design#39831] [Show and tell]
Well I'm new here and I'm looking for a solution to a problem with the Popover component. I recently updated antD to version 5.1.1 and noticed this behavior in Popover. Even using the placement parameter, dont matter if trigger is click or hover, when displaying the box, it always contains 2 arrows, the one determined in the placement and one always pointing upwards.

Here is the image with the problem...
<img width="224" alt="bug_popover" src="https://user-images.githubusercontent.com/32021798/209572776-13986c7c-c85b-4308-8038-3968209d1aac.png">

Here is the component call...
<img width="233" alt="bug_popover2" src="https://user-images.githubusercontent.com/32021798/209572877-6f1a5e94-639f-4d43-a670-4dfeae6f08ae.png">

Someone have some ideia why it happen? 
Thanks.

**Comment by MadCcc** [comment]
The style is mixed up with antd 4.x. Please check if there is 4.x version in your project.
And the solution is change the `prefixCls` of antd 5.x or remove antd 4.x.

**Reply by Orkuml** [reply]
Hello madCcc, thanks for the answer. I really appreciate.
Well, based on your answer, find the problem. I use theme.css (global Theme for ant) and this caused the problem.
I removed and popover work normally again.
Thanks. <3
